components/admin_console/admin_console.tsx

```diff
@@ -241,6 +242,7 @@ export default class AdminConsole extends React.PureComponent<Props, State> {
                 <BackstageNavbar
                     team={this.props.team}
                 />
+                <DelinquencyModal/>
```

Comment:
Obviously I'm late on this, but this component shouldn't need to be mounted here since this modal is being controlled by the `ModalController` (ie using the `openModal`/`closeModal` actions). Dispatching `openModal` has it mounted by the `ModalController` automatically

Reply:
@Afsoon heads up on the above - can you create a follow up ticket for this?